examples/trading/frontend/src/components/ObjectLink.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useSuiClientContext } from "@mysten/dapp-kit";
+import { formatAddress } from "@mysten/sui.js/utils";
+import { CheckIcon, CopyIcon } from "@radix-ui/react-icons";
+import { useState } from "react";
+import toast from "react-hot-toast";
+
+export function ObjectLink({
```

Comment:
nit: given this does both object and addresses, I would call this an `ExplorerLink` rather than an `ObjectLink`.